packages/storybook-blocks/src/components/status-tag/status-tag.module.scss

```diff
@@ -4,7 +4,7 @@
   padding: 0 var(--sb-spacing-small);
   text-transform: capitalize;
 
-  &.beta {
+  &.alpha, &.beta {
```

Comment:
Do we want it with same style on purpose?

Reply:
Yep, thats what it said in the task :)

Reply:
Perfect 👌🏻